Week_2/Week_2/Challenge_2-1.py

```diff
@@ -0,0 +1,8 @@
+def print_numbers():
+    num = 0

```

Comment:
Why do you need this variable?

Reply:
I thought I had to initialize the variable before adding the increments.